File: ru.kmorozov.gbd.core.logger/src/main/java/ru/kmorozov/gbd/logger/output/IOutputReceiver.java
package ru.kmorozov.gbd.logger.output;

import ru.kmorozov.gbd.logger.model.ILoggableObject;

/**
 * Created by km on 13.12.2015.
 */
@FunctionalInterface
public interface IOutputReceiver {

    void receive(ILoggableObject bookInfo);
}

File: ru.kmorozov.gbd.core.logger/src/main/java/ru/kmorozov/gbd/logger/output/IOutputReceiver.kt
package ru.kmorozov.gbd.logger.output

import ru.kmorozov.gbd.logger.model.ILoggableObject

/**
 * Created by km on 13.12.2015.
 */
@FunctionalInterface
interface IOutputReceiver {

    fun receive(bookInfo: ILoggableObject)
}
